tests/core/test_playback.py

```diff
@@ -124,6 +125,22 @@ def test_play_skips_to_next_on_unplayable_track(self):
         self.assertEqual(
             self.core.playback.current_tl_track, self.tl_tracks[3])
 
+    def test_play_skips_to_next_on_unplayable_track(self):
+        """Checks that we handle change track failing."""
+        self.playback2.change_track().get.return_value = False
```

Comment:
We do this wrong all over the place, and thus we often need to use `reset_mock()` in the beginning of tests.

The proper way is to replace `()` with `.return_value`.